.github/workflows/php.yml

```diff
@@ -0,0 +1,36 @@
+name: PHP Composer
+
+on:
+  release:
+    types: [published]
+
+jobs:
```

Comment:
Can you add a new line after this?
Similar to the YAML files in the other repos